homeassistant/components/ipma/weather.py

```diff
@@ -103,6 +122,23 @@ def __init__(self, station, config):
             self._forecast = await self._station.forecast()
             self._description = self._forecast[0].description
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique id."""
+        return '{}, {}'.format(self._station.latitude, self._station.longitude)
+
+    @property
+    def device_info(self):
+        """Return device information."""
+        return {
+            'identifiers': {
+                (DOMAIN, self.unique_id)
+            },
+            'name': self.name,
+            'manufacturer': "Instituto Português do Mar e Atmosfera",
+            'model': "{} weather station".format(self._station.local),
```

Comment:
This is a service, and not a device, right?

Reply:
see above